allure-report-face/src/components/widget-overview/OverviewWidget.js

```diff
@@ -1,7 +1,53 @@
 import './styles.css';
 import {ItemView} from 'backbone.marionette';
 import template from './OverviewWidget.hbs';
+import c3 from 'c3';
```

Comment:
When we will be ready to start optimizing this report, i'll rewrite this to raw d3. For now its simpler to use very simple configuration of c3 with sweet style :)